### DISCUSSION TITLE
Reverse include for one to one relationship?

### DISCUSSION THREAD

**Discussion by bizk-sato** [prisma/prisma#13863] [Q&A]
Let's say if we have User and Profile model(User has one Profile).

```
model User {
  id                    Int      @id @default(autoincrement())
  profile Profile?

model Profile {
  id int @id @default(autoincrement())
  userId   Int?                                              
  user     User? @relation(fields: [userId], references: [id])
}
```

And I want to get list of users with a profile if they they have.
So I tried to query like below but it throws error.

```
 prisma.user.findMany({ include: { profile: true })
```

The error message was `The GraphQL context.prisma value is not an instance of the Prisma Client`

Is there any way to get users with a profile?

**Comment by nurul3101** [comment]
Hey @bizk-sato 👋 
How are you passing the `PrismaClient` instance to the GraphQL context?

Here's an example of how you could do it:

```typescript
const prisma = new PrismaClient()

export async function contextResolver(ctx: Ctx): Promise<Ctx> {
  ctx.db = prisma
}
```

**Comment by bizk-sato** [comment]
@nurul3101 

Hi, I'm using apollo and passing the client is something like below.

Normal one to many `include` works but only the one to one reverse `include` does not.

```
 const prisma= new PrismaClient({
    log: ["query"],
  });

const apolloServer = new ApolloServer({ schema, context: { prsima, req, res } })
```

**Reply by pantharshit00** [reply]
Hello @bizk-sato , our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

If this is still relevant, can you please share a reproduction? I am unable to reproduce this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
